hack/testing/test-010-deploy-via-olm-minimal.sh

```diff
@@ -22,7 +23,7 @@ version=$(basename $(find $manifest -type d | sort -r | head -n 1))
 cleanup(){
   local return_code="$?"
   set +e
-  oc logs -n ${NAMESPACE} deployment/cluster-logging-operator > $ARTIFACT_DIR/cluster-logging-operator.log 2>&1 ||:
+  gather_logging_resources ${NAMESPACE} $test_artifactdir
```

Comment:
The lines after gather_logging_resources are executed with `set -e` whereas in the old code they were executed with `set +e`. Not sure if this is intentional or relevant?